ui/components/form/ConfigForm.tsx

```
@@ -84,6 +324,149 @@ export default function ConfigForm({ onSubmit, defaultConfig }: ConfigFormProps)
+                        {error && <p className="mt-1 text-xs text-red-600">{error}</p>}
+                      </div>
+                    );
+                  }
+
+                  if (field.kind === 'string_enum') {
+                    const value = typeof feedAlgorithmConfig[field.key] === 'string' ? (feedAlgorithmConfig[field.key] as string) : '';
+                    return (
+                      <div key={field.key}>
+                        <label className="block text-sm font-medium text-beige-800 mb-2">
+                          {field.label}
+                        </label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)